charts/promitor-agent-scraper/values.yaml

```diff
@@ -8,7 +8,7 @@ fullnameOverride: ""
 
 image:
   repository: tomkerkhove/promitor-agent-scraper
-  tag: 1.0.0
+  tag: latest
```

Comment:
Can you revert this please? AFAIK this is being used for find/replace; but could be wrong.

Reply:
See https://github.com/tomkerkhove/promitor/blob/master/build/helm/CI/Transform-Chart.ps1#L21